definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-alt-service-process-server-nonprod.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "LiveFrom": "05/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "ProcessServerServiceCaseField",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "05/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "ProcessServerServiceCaseField",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "05/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "ProcessServerServiceCaseField",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "05/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "ProcessServerServiceCaseField",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "05/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "ProcessServerServiceCaseField",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "05/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "ProcessServerServiceCaseField",
```

Comment:
Is this field going to be set to "Yes" by the COS endpoint?

Reply:
Correct, Ive changed the name to ProcessServerServiceDecision

Reply:
So is that PR going to be developed as part of this story?